### PR TITLE
MAYA-106274 - Missing temp for "mayaUsdProxyShapeBase" node in AE

### PR DESCRIPTION
**Pull request by seando-adsk**
* Created AE template for mayaUsdProxyShapeBase by moving everything
  from existing mayaUsdProxyShape template. Then for mayaUsdProxyShape
  template just include the base.

### CODE REVIEW

File: plugin/adsk/scripts/AEmayaUsdProxyShapeBaseTemplate.mel [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+    // Call python helpers to get the info we want to display.
+    string $rootLayer   = `python("import AETemplateHelpers; AETemplateHelpers.GetRootLayerName('" + $fullNodeName + "')")`;
+    string $defaultPrim = `python("import AETemplateHelpers; AETemplateHelpers.GetDefaultPrimName('" + $fullNodeName + "')")`;
+
+    textFieldGrp -edit -text $rootLayer mayaUsdProxyShapeRootLayer;
+    textFieldGrp -edit -text $defaultPrim mayaUsdProxyShapeDefaultPrim;
+}
+
+global proc AEmayaUsdProxyShapeBaseTemplate( string $nodeName )
```
Comment (seando-adsk) [inline plugin/adsk/scripts/AEmayaUsdProxyShapeBaseTemplate.mel:29]: I created this file by copying the existing "AEmayaUsdProxyShapeTemplate.mel" and renaming this proc (to match the proxy shape base node).

File: plugin/adsk/scripts/AEmayaUsdProxyShapeTemplate.mel [hunk 2]
```diff
@@ -1,249 +1,5 @@
...
-        editorTemplate -suppress "collisionOffsetVelocityIncrement";
-        editorTemplate -suppress "collisionDepthVelocityIncrement";
-        editorTemplate -suppress "collisionOffsetVelocityMultiplier";
-        editorTemplate -suppress "collisionDepthVelocityMultiplier";
-
-        editorTemplate -addExtraControls;
-
-    editorTemplate -endScrollLayout;
+	// Just include our base template - it has all the attributes.
+	AEmayaUsdProxyShapeBaseTemplate $nodeName;
```
Comment (seando-adsk) [inline plugin/adsk/scripts/AEmayaUsdProxyShapeTemplate.mel:4]: The template for our proxy shape node just calls the base.

**Review by fowlertADSK — APPROVED** [review]
LGTM

**Comment by seando-adsk** [comment]
@kxl-adsk This one is ready for merge. I don't see any point to run the preflight as what I've changed is the AE template for the proxy shape.

**Pull request merged**